### PR TITLE
Add `define-data-property` and `define-accessor-property`

### PR DESCRIPTION
**Pull request by zloirock**

### CODE REVIEW

File: manifests/micro-utilities.json [hunk 1]
```diff
@@ -6,6 +6,12 @@
       "replacement": "Use Function.call.bind(v)",
       "category": "micro-utilities"
     },
+    {
+      "type": "simple",
+      "moduleName": "define-data-property",
+      "replacement": "Use Object.defineProperty(object, key, { value, enumerable, configurable, writable })",
```
Comment (zloirock) [inline manifests/micro-utilities.json:12]: Maybe, it should be in the `native` manifest. WDYT?

Comment (43081j) [reply manifests/micro-utilities.json:12]: Since it is a one to one replacement, I think so

If it was more ambiguous, it would belong here

Comment (zloirock) [reply manifests/micro-utilities.json:12]: This util works only with the creation of data properties, `Object.defineProperty` also can be used to change descriptors and create accessors.

Comment (zloirock) [reply manifests/micro-utilities.json:12]: But yes, `Object.defineProperty` is one to one replacement, just with another signature.

Comment (zloirock) [reply manifests/micro-utilities.json]: Updated.

**Review by 43081j — APPROVED** [review]

**Pull request merged**